_includes/comparison.html

```diff
@@ -250,7 +250,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td><a href="https://istio.io/docs/setup/install/multicluster/" target="_blank">yes</a></td>
         <td><a href="https://linkerd.io/2/features/multicluster/" target="_blank">yes</a></td>
         <td></td>
-        <td><a href="https://www.consul.io/docs/k8s/deployment-configurations/multi-cluster" target="_blank">yes</a></td>
+        <td><a href="https://developer.hashicorp.com/consul/docs/connect/cluster-peering" target="_blank">yes</a></td>
```

Comment:
I think this is a description of a different feature, not multi-cluster support. Also the old link still works. I am therefore unsure why you submitted this change?

Reply:
Hi @ewolff Cluster Peering is our new model for federating multiple Consul clusters together that run on separate K8s clusters. Our WAN Fed model will likely be deprecated in the future but still exists in that link that was perviously provided. We will likely make changes to reflect that in our docs information architecture but is not updated yet. You can see more details on how this works here: https://developer.hashicorp.com/consul/docs/connect/cluster-peering/k8s

Reply:
Great, thanks a lot for explaining! 🙂👍